crates/sql-sink/src/db.rs

```diff
@@ -288,13 +300,30 @@ mod tests {
         }
     }
 
-    fn check_row(row: AnyRow) {
+    fn check_row<'r, DB, T>(row: &'r T)
```

Comment:
I am removing AnyRow here, because it's lacking some types in the new versions of sqlx. So, I am using the native types now.

https://github.com/launchbadge/sqlx/issues/2778